include/kernels/color_spinor_pack.cuh

```diff
@@ -78,9 +80,22 @@ namespace quda {
     int_fastdiv work_items;
     int threadDimMapLower[4];
     int threadDimMapUpper[4];
+#ifdef NVSHMEM_COMMS
+    char *packBuffer[4 * QUDA_MAX_DIM];
```

Comment:
I don't think `packBuffer` is used anywhere.

Reply:
This is used through `shame_pack_helper.cuh`